app/assets/javascripts/discourse/app/routes/users.js

```diff
@@ -36,11 +37,16 @@ export default DiscourseRoute.extend({
   },
 
   model(params) {
-    return params;
+    const columns = PreloadStore.get("directoryColumns");
+    if (!params.order) {
+      params.order = columns[0].name;
```

Comment:
Trivial, but this if block could be one line: `params.order = params.order || columns[0].name;`